lib/lita/message.rb

```diff
@@ -32,10 +37,11 @@ class Message
     # @param robot [Robot] The currently running robot.
     # @param body [String] The body of the message.
     # @param source [Source] The source of the message.
-    def initialize(robot, body, source)
+    def initialize(robot, body, source, adapter_data = {})
```

Comment:
Don't change the constructor. Just initialize the ivar to an empty hash. Users will access it through the attr_reader.